src/crimson/os/seastore/seastore.h

```diff
@@ -208,7 +223,9 @@ class SeaStore final : public FuturizedStore {
 	  }).safe_then([&ret](auto _ret) {
 	    ret = _ret;
 	  });
-	}).safe_then([&ret] {
+	}).safe_then([&ret, optype, &t, this]{
+          const_cast<SeaStore*>(this)->inc_lat(optype,
+                                std::chrono::steady_clock::now() - t->tp);
```

Comment:
`create_transaction()` is inside the `repeat_eagain()` loop, each reset will create a fresh transaction and reset the timestamp. Move `tp` out of the transaction if want to measure latency at  operation level .